.github/workflows/notes.md

```diff
@@ -34,25 +34,43 @@ Note: On Windows "Server" you may need to [install vcruntime140.dll](https://lea
 3. Extract: `tar xzvf nvim-macos-arm64.tar.gz`
 4. Run `./nvim-macos-arm64/bin/nvim`
 
-### Linux (x64)
+### Linux (x86_64)
 
 glibc 2.31 or newer is required. Or you may try the (unsupported) [builds for older glibc](https://github.com/neovim/neovim-releases).
 
 #### AppImage
 
-1. Download **nvim.appimage**
-2. Run `chmod u+x nvim.appimage && ./nvim.appimage`
+1. Download **nvim-linux-x86_64.appimage**
+2. Run `chmod u+x nvim-linux-x86_64.appimage && ./nvim-linux-x86_64.appimage`
    - If your system does not have FUSE you can [extract the appimage](https://github.com/AppImage/AppImageKit/wiki/FUSE#type-2-appimage):
      ```
-     ./nvim.appimage --appimage-extract
+     ./nvim-linux-x86_64.appimage --appimage-extract
      ./squashfs-root/usr/bin/nvim
      ```
 
 #### Tarball
 
-1. Download **nvim-linux64.tar.gz**
-2. Extract: `tar xzvf nvim-linux64.tar.gz`
-3. Run `./nvim-linux64/bin/nvim`
+1. Download **nvim-linux-x86_64.tar.gz**
+2. Extract: `tar xzvf nvim-linux-x86_64.tar.gz`
+3. Run `./nvim-linux-x86_64/bin/nvim`
+
+### Linux (aarch64)
```

Comment:
for the macos builds we call them `macos-arm64`, wouldn't it make sense to be consistent with how we describe "arm architecture"?

Reply:
See above. There's a technical reason for that.